services/workflows-service/src/customer/api-key/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { env } from '@/env';
+import { faker } from '@faker-js/faker';
+import { ApiKeyType } from '@prisma/client';
+import * as crypto from 'crypto';
+
+const ONE_DAY_IN_MS = 1000 * 60 * 60 * 24;
+const PASSWORD_REGEX = /[!@#$%^&*a-zA-Z]/;
+const API_KEY_LEN = 50;
+
+export const encryptApiKey = (apiKey: string) => {
```

Comment:
It's important to use the right term here.
```suggestion
export const hashApiKey = (apiKey: string) => {
```